docs/ServiceNowMultiSource-batchsource.md

```diff
@@ -22,6 +22,10 @@ Properties
 
 **Password**: The password for ServiceNow Instance.
 
+**Use Connection:** Whether to use a connection. If a connection is used, you do not need to provide the credentials.
```

Comment:
Can you make sure the order of the properties matches the UI? Usually Connection properties appear near the top of the list of properties in the UI...

Reply:
Changed